CSLabs.Api/Migrations/20210809190958_PermanentLabLimit.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace CSLabs.Api.Migrations
+{
+    public partial class PermanentLabLimit : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
```

Comment:
Where's the migration code? The `migrations add` command should have generated the statements for you.